packages/terra-clinical-application-site/src/index.scss

```diff
@@ -16,6 +16,7 @@ body {
   height: 100%;
 }
 
-.terraClinical-ItemView-accessory {
+[class*="ItemView__accessory"],
+[class*="ItemView__accessory"] > * {
```

Comment:
I agree, it seems like a tight constraint and it is brittle css. This is something Tim & I discussed at on point on the requirements side when I was working on item-collection.